tests/integration/targets/setup_mysql_db/tasks/main.yml

```
@@ -63,6 +63,13 @@
   when: ansible_facts.pkg_mgr == 'apt'
   notify: cleanup mysql
 
+- name: install mysqldb_test opensuse dependencies
+  zypper:
+    name: "{{ mysql_packages }}"
+    state: latest
+  when: ansible_facts.pkg_mgr == 'zypper'
```

For now (until ansible/ansible#67684 is merged), this needs to be:
```suggestion
  when: ansible_facts.pkg_mgr in ['zypper', 'community.general.zypper']
```
(See also #60)